mini-cap/src/components/Popup_SendKey.js

```diff
@@ -2,13 +2,14 @@ import React, { useState } from "react";
 import "../styling/Popup.css";
 import "../index.css";
 import { toast } from "react-toastify";
-import {addUser, checkEmailExists} from "../backend/Fetcher";
-
-const Popup_SendKey = ({ handleClose }) => {
+import {storeCondoKey, checkEmailExists} from "../backend/Fetcher";
 
+const Popup_SendKey = ({ handleClose, condoId }) => {
+    const [showPopup, setShowPopup] = useState(true);
     const [formData, setFormData] = useState({
-        role: "renter", // "Renter" as the default role
+        role: "Renter", //default for now
```

Comment:
We should use either "Renter" and "Owner" or "renter" and "owner". Line 68 uses lower case so maybe switch this one to lowercase